Covid19Radar/Covid19Radar.Android/Services/DeviceCheckService.cs

```diff
@@ -24,6 +26,26 @@ public Task<string> VerifyAsync(V1EventLogRequest eventLogRequest)
             return GetSafetyNetAttestationAsync(nonce);
         }
 
+        public bool IsErrorPayload(string jwsResult)
```

Comment:
インターフェースと引数名が異なるのが気になります。

```suggestion
        public bool IsErrorPayload(string payload)
```

Reply:
実装の過程で変数名見直して中途半端になってしまったものですね。修正します。

Reply:
パラメータ名を修正しました

Reply:
さらにパラメータ名を修正しました

Reply:
OK です！